warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__feed_present_service_alerts.sql

```diff
@@ -0,0 +1,38 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__rt_feed_guideline_index') }}
+     WHERE feed_type = 'service_alerts'
+),
+
+fct_daily_rt_feed_files AS (
```

Comment:
Sorry to keep doing this (nuance/confusion about indexing on feeds/files vs. URLs/dataset records in Airtable), but in #2010 I have a daily URLs list for RT which might be better than checking files -- daily URLs will say whether we were *attempting* to hit a service alerts feed. Issue with `fct_daily_rt_feed_files` is that feeds with 0 files will not be listed, but if this is just checking for existence of a feed I think that the list of URLs is better.

Reply:
But defer to you if you only want to give credit for feeds where we can get at least one file, I think that could make sense; just not sure where this check is meant to fall on the spectrum of success/validity vs. pure existence.